frontend/cypress/integration/components/mailingForm.spec.js

```diff
@@ -0,0 +1,70 @@
+describe('Mailing forms', () => {
+  it('checks labels and validation of a general form', () => {
+    // Visit engage page
+    cy.visit('/#/engage');
+    // Select general form tab
+    cy.get('[data-cy=general-form-tab]').click();
+    // Check if name label exists
+    cy.get('[data-cy=general-name-label]').contains('Name *');
+    // Check if email label exists
+    cy.get('[data-cy=general-email-label]').contains('Email *');
+    // Check if message label exists
+    cy.get('[data-cy=general-message-label]').contains('Message *');
+    // Ensure send button is disabled
+    cy.get('[data-cy=general-send-button]').should('be.disabled');
+    // Fill in valid name
+    cy.get('[data-cy=general-name-field]').type('Sergio');
+    // Fill in valid email
+    cy.get('[data-cy=general-email-field]').type('sergio@gmail.com');
+    // Fill in valid message
+    cy.get('[data-cy=general-message-field]').type('message goes here');
+    // Ensure send button is enabled after entering valid inputs
+    cy.get('[data-cy=general-send-button]').should('not.be.disabled');
+  });
+
+  it('checks labels and validation of a sponsorship form', () => {
+    // Visit sponsors page
+    cy.visit('/#/sponsors');
+    // Check if name label exists
+    cy.get('[data-cy=sponsorship-name-label]').contains('Company Name *');
+    // Check if email label exists
+    cy.get('[data-cy=sponsorship-email-label]').contains('Email *');
+    // Check if message label exists
+    cy.get('[data-cy=sponsorship-message-label]').contains('Message *');
+    // Ensure send button is disabled
+    cy.get('[data-cy=sponsorship-send-button]').should('be.disabled');
```

Comment:
You've already checked this. Because its part of one component there should not be the need to do it twice, more so that the Mailing component is actually there.